.github/workflows/wordpress-plugin-asset-update.yml

```diff
@@ -2,16 +2,17 @@ name: Plugin asset/readme update
 on:
   push:
     branches:
```

Comment:
We obviously habe to rename the branches before this works.

Reply:
My plan is to create the `develop` branch directly after this PR is merged and `stable` when the next release is ready.